packages/web/src/fixtures/dev-kit/cache-path.ts

```diff
@@ -5,9 +5,8 @@ export const SWRCachePath = {
   getMyStakingRewardAmount: (propertyAddress: string) =>
     `propertyAddresses/${propertyAddress}/getMyStakingRewardAmount`,
   getMyStakingAmount: (propertyAddress: string) => `propertyAddresses/${propertyAddress}/getMyStakingAmount`,
-  getLastAssetValueEachMetrics: (metricsAddress: string) =>
-    `metricsAddress/${metricsAddress}/getLastAssetValueEachMetrics`,
-  getLastAssetValueEachMarketPerBlock: (marketAddress: string) =>
-    `marketAddress/${marketAddress}/getLastAssetValueEachMarketPerBlock`,
-  getWithdrawalStatus: (propertyAddress: string) => `propertyAddresses/${propertyAddress}/getWithdrawalStatus`
+  getTotalStakingAmountOnProtocol: () => `getTotalStakingAmountOnProtocol`,
+  getWithdrawalStatus: (propertyAddress: string) => `propertyAddresses/${propertyAddress}/getWithdrawalStatus`,
+  calculateMaxRewardsPerBlock: () => `calculateMaxRewardsPerBlock`,
```

Comment:
If you don't want to change the cache path dynamically, it's better to define it as a normal string !

Reply:
Ah, okay, thank you for your review!